pilot/pkg/bootstrap/istio_ca.go

```diff
@@ -285,6 +290,42 @@ func (s *Server) initPublicKey() error {
 	return nil
 }
 
+// loadRemoteCACerts mounts an existing cacerts Secret if the files aren't mounted locally.
+// By default, a cacerts Secret would be mounted during pod startup due to the
+// Istiod Deployment configuration. But with external Istiod, we want to be
+// able to load cacerts from a remote cluster instead.
+func (s *Server) loadRemoteCACerts(caOpts *CAOptions, dir string) error {
+	if s.kubeClient == nil {
+		return nil
+	}
+
+	signingKeyFile := path.Join(dir, "ca-key.pem")
```

Comment:
should we check all the files? or was there a reason we chose just `ca-key`?

Reply:
Yeah it doesn't feel ideal; other places in the code checked only this one, so this was for consistency